map/src/data/markers.ts

```diff
@@ -62,6 +62,32 @@ export interface MarkerInfo {
 }
 
 const LOCATIONS = {
+  PT: {
+    ALMADA: {
+      description: 'Almada, Portugal',
+      lat: 38.678506,
+      lng: -9.164985,
+      serviceRadius: 5000,
+    },
+    LISBON_BELEM: {
+      description: 'Lisbon, Belém, Portugal',
+      lat: 38.7026973,
+      lng: -9.2089478,
+      serviceRadius: 2000,
+    },
+    PORTO_FOZ_DO_DOURO: {
+      description: 'Porto, Foz do Douro, Portugal',
+      lat: 41.1675792,
+      lng: -8.7027557,
+      serviceRadius: 2000,
+    },
+    ALL: {
+      description: 'PT',
+      lat: 37.1458994,
+      lng: -23.3165484,
+      serviceRadius: 280610,
```

Comment:
```suggestion
      description: 'PT',
      lat: 39.475803,
      lng: -8.591411,
      serviceRadius: 311000,
```

Reply:
Unless you want to explicitly include the islands and not the mainland, in which case the radius needs to be a lot bigger :)